### PR TITLE
Converting PFS refucer tests to new pattern

### LINKED ISSUE #1616: Convert services unit tests to the new pattern
## Description
With #1377 / #1577 , there's a new standardized way to do epics unit async testing.
We should convert the services (path, monitoring) tests to use this new pattern. Check #1614 for the new flow. We should also convert the generic unit `raiden.spec.ts` on this issue.
New `ensure*` fixtures functions may be needed to ensure some common state.

## Acceptance criteria
- [ ] [path.spec.ts](https://github.com/raiden-network/light-client/blob/master/raiden-ts/tests/unit/epics/path.spec.ts) is converted to the new tests fixtures
- [x] [monitor.spec.ts](https://github.com/raiden-network/light-client/blob/master/raiden-ts/tests/unit/epics/monitor.spec.ts) is converted to the new tests fixtures
- [x] [raiden.spec.ts](https://github.com/raiden-network/light-client/blob/master/raiden-ts/tests/unit/epics/raiden.spec.ts) is converted to the new tests fixtures
- [ ] Tests don't use neither old `raidenEpicDeps` nor `epicFixtures` utilities, and instead rely on the new fixtures: `makeRaiden`, `makeRaidens`, `ensure*` and constants

### PR DESCRIPTION
**Pull request by agatsoh**
**Thank you for submitting this pull request :)**

Fixes #1616 

**Short description**

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: raiden-ts/tests/unit/fixtures.ts [hunk 1]
```diff
@@ -166,6 +166,22 @@ export const secret = makeSecret();
 export const secrethash = getSecrethash(secret);
 export const amount = bigNumberify(10) as UInt<32>;
 export const fee = bigNumberify(3) as Int<32>;
+export const pfsAddress = makeAddress();
+
+/**
+ * @param raiden - Instance of MockedRaiden
+ * @returns Mocked IOU type Object
+ */
+export function makeIou(raiden: MockedRaiden): IOU {
```
Comment (andrevmatos) [inline raiden-ts/tests/unit/fixtures.ts:175]: This function is only relevant for this specific test suite. Please, move it to the top of the `path.spec.ts` file, no need for it to be in the shared fixtures.

Comment (agatsoh) [reply raiden-ts/tests/unit/fixtures.ts:175]: ok sure will move it at the top of this test suite

**Comment by agatsoh** [comment]
@andrevmatos you could have a look once more after the tests are done.

**Review by andrevmatos — APPROVED** [review]
lgtm, ty

**Pull request merged**